backend/src/entity/Download.ts

```diff
@@ -21,14 +21,18 @@ export class Download {
   @Column()
   ip!: string
 
+  @Column({ type: 'char', length: 2, nullable: true })
```

Comment:
Onko aina 2 merkkiä?

Reply:
Dokumentaation mukaan on: https://dev.maxmind.com/geoip/docs/databases/city-and-country